pentasharp/Models/Entities/User.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace pentasharp.Models.Entities
+{
+    public class User
+    {
+        public int UserId { get; set; } 
+        public string FirstName { get; set; } = string.Empty;
+        public string LastName { get; set; } = string.Empty;
+        public string Email { get; set; } = string.Empty;
+        public string PasswordHash { get; set; } = string.Empty;
+        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
+        public string Role { get; set; } = string.Empty;
```

Comment:
If Role is a specific value: user or admin, create an enum called UserRole that will have those specific values and then fix this property to public UserRole Role {get; set;}